gensim/models/ldaseqmodel.py

```diff
@@ -0,0 +1,1121 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+# Based on Copyright (C) 2016 Radim Rehurek <radimrehurek@seznam.cz>
+
+
+"""
+
+This is the class which is used to help with Dynamic Topic Modelling of a corpus.
+It is a work in progress and will change largely throughout the course of development.
+Inspired by the Blei's original DTM code and paper. TODO: add links
+
+As of now, the LdaSeqModel and SSLM classes mimic the structures of the same name in the Blei DTM code.
+Few mathematical helper functions will be made and tested.
+
+"""
+
+from gensim import interfaces, utils, matutils
+from gensim.models import ldamodel
+import numpy
+import math
+from scipy.special import digamma
+from scipy import optimize
+
+
+class seq_corpus(utils.SaveLoad):
+
+    """
+    seq_corpus is basically a wrapper class which contains information about the corpus.
+    num_terms is the length of the vocabulary.
+    max_nterms is the maximum number of terms a single document has.
+    num_sequences is the number of sequences, i.e number of time-slices.
+    num_docs is the number of documents present.
+    time_slice is a list or numpy array which the user must provide which contains the number of documents in each time-slice.
+    corpus is any iterable gensim corpus.
```

Comment:
can you change names to `vocab_len`, `max_doc_len`, `num_time_slices`, `corpus_len` ?